projects/plugins/crm/modules/woo-sync/includes/class-woo-sync-background-sync-job.php

```diff
@@ -892,7 +892,7 @@ public function import_crm_object_data( $crm_object_data ) {
 
 				} else {
 
-						$this->debug( 'Company import failed: <code>' . json_encode( $crm_object_data['company'] ) . '</code>' );
+						$this->debug( 'Company import failed: <code>' . wp_json_encode( $crm_object_data['company'], JSON_UNESCAPED_SLASHES | JSON_HEX_TAG | JSON_HEX_AMP ) . '</code>' );
```

Comment:
Similar to the above, the output will probably be nicer if you do `esc_html()` or the like instead of `JSON_HEX_TAG`.

Assuming this even really outputs HTML at all? It looks like it might be a CLI job?

Same for the other instances later in this file.